scripts/docker-compose.yml

```
@@ -79,38 +80,69 @@ services:
       POSTGRES_USER: insights
       POSTGRES_DB: insights
     ports:
-        - "15433:5432"
-  insights-inventory:
+      - "15433:5432"
+
+  insights-inventory-mq: &inventory
     image: quay.io/cloudservices/insights-inventory:latest
     restart: always
-    ports:
-      - 8081:8081
+    command: "make upgrade_db run_inv_mq_service"
     environment:
-      - APP_FILE=run_gunicorn.py
       - APP_NAME=inventory
       - PATH_PREFIX=api
       - INVENTORY_DB_USER=insights
       - INVENTORY_DB_PASS=insights
       - INVENTORY_DB_HOST=db-host-inventory
       - INVENTORY_DB_NAME=insights
       - KAFKA_BOOTSTRAP_SERVERS=kafka:29092
+      - KAFKA_TOPIC=platform.system-profile
+      - KAFKA_GROUP=inventory
       - PAYLOAD_TRACKER_ENABLED=false
       - LISTEN_PORT=8081
     depends_on:
        - kafka
        - db-host-inventory
        - puptoo
-       - zookeeper
+
+  insights-inventory-web:
+    <<: *inventory
+    command: "make upgrade_db run_inv_web_service"
+    depends_on:
+      - insights-inventory-mq
+    ports:
+      - 8081:8080
+
   puptoo:
     image: quay.io/cloudservices/insights-puptoo:latest
     ports:
       - 8000:8000
     environment:
-      - LOG_LEVEL=DEBUG
-      - ACG_CONFIG=/cdappconfig.json
-      - INVENTORY_TOPIC=platform.inventory.host-ingress
+      - LOG_LEVEL=INFO
+      - FACT_EXTRACT_LOGLEVEL=ERROR
+      - KAFKA_ALLOW_CREATE_TOPICS=True
+      - DISABLE_PROMETHEUS=True
     depends_on:
       - kafka
       - ingress
+    # Not yet used, we are not running clowder ATM
     volumes:
-        - '~/cdappconfig.json:/cdappconfig.json'
\ No newline at end of file
+      - './cdappconfig.json:/cdappconfig.json:Z'
+
+  # Allows us to run the application in the form it's going to be deployed, will be needed for tests
+  ros-processor: &ros
+    build:
+      context: ..
+    command: [ "python", "-m", "ros.processor.main" ]
+    depends_on:
+      - insights-inventory-web
+      - kafka
+      - createbuckets
+    environment:
+      - INSIGHTS_KAFKA_HOST=kafka
+      - INSIGHTS_KAFKA_PORT=29092
```

```suggestion
      - INSIGHTS_KAFKA_PORT=9092
```
As per previous changes made, in the ROS config file & here, modified port to `9092`. At this line also will it be good to have `9092`?

---

When running inside docker, we need to set different port. The kafka container listens on both, but kafka consists of several parts. What we contact at first is the broker